src/main/java/org/jabref/logic/importer/fileformat/WorldcatImporter.java

```diff
@@ -0,0 +1,175 @@
+package org.jabref.logic.importer.fileformat;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.StringReader;
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.xml.parsers.DocumentBuilder;
+import javax.xml.parsers.DocumentBuilderFactory;
+import javax.xml.parsers.ParserConfigurationException;
+
+import org.jabref.logic.importer.Importer;
+import org.jabref.logic.importer.ParserResult;
+import org.jabref.logic.importer.fetcher.WorldcatFetcher;
+import org.jabref.logic.net.URLDownload;
+import org.jabref.logic.util.FileType;
+import org.jabref.logic.util.StandardFileType;
+import org.jabref.model.entry.BibEntry;
+import org.jabref.model.entry.field.StandardField;
+
+import org.w3c.dom.Document;
+import org.w3c.dom.Element;
+import org.w3c.dom.NodeList;
+import org.xml.sax.InputSource;
+import org.xml.sax.SAXException;
+
+public class WorldcatImporter extends Importer {
+
+	private static final String DESCRIPTION = "Takes valid XML from Worldcat Open Search and parses them to BibEntry";
```

Comment:
```suggestion
	private static final String DESCRIPTION = "Importer for Worldcat Open Search XML format";
```
Made it a bit consistent to the other importers.